test/hl/common.py

```diff
@@ -201,23 +226,32 @@ def assertNumpyBehavior(self, dset, arr, s):
             with self.assertRaises(exc):
                 dset[s]
 
-    def getFileName(self, basename):
+    def getFileName(self, basename, subfolder=None):
         """
         Get filepath for a test case given a testname
         """
 
+        # Just call the external function
+        filename = getTestFileName(basename, subfolder=subfolder)
```

Comment:
`getFileName` invokes `getTestFileName`, but then overwrites the result to redo the same work.

Reply:
I made the change so getTestFileName can be invoked outside the TestCase class, but forgot to remove the duplicate code.  Fixed now.